### ISSUE TITLE
Question about network interpolation

### ISSUE THREAD

**Issue by zengwei1302** [xinntao/BasicSR#149]
After the network interpolation, the test results become very dark. 
Both models tested well before the network interpolation.
So I want to ask where the problem might be.

**Comment by xinntao** [comment]
I have also encountered this problem. This is because when fine-tuning, the fine-tuned network is **too far** from the pretrained model. It would be very interesting to define network distance and then restrict it. And we are still exploring it. 

You can try 1) using a smaller learning rate to fine-tune. 2) add a residual path in the network, so the network just learns the residual. These two methods can help to alleviate this problem in my practice. They are both for not pushing the two network too far.

**Comment by victorca25** [comment]
If the second network was trained from scratch, you will always get dark results after inteprolating

**Comment by aldogan17** [comment]
Hi all, I couldn't run network interpolation script. Somehow my trained esrgan and psnr models from scratch are not accepted. Because when I do the same thing for pretrained models from google drive repo, it works. What is the catch?
